ipaplatform/debian/tasks.py

```diff
@@ -206,4 +202,10 @@ def remove_ca_certificates_dir(self, directory):
 
         return True
 
+    def enable_ldap_automount(self, statestore):
+        return BaseTaskNamespace.enable_ldap_automount(self, statestore)
```

Comment:
Could you please add a comment here to explain why you are calling the methods from the base namespace?